src/state.rs

```diff
@@ -191,6 +188,25 @@ impl State {
         Ok(())
     }
 
+    pub fn set_active_capella_org_id(&mut self, id: String) -> Result<(), ShellError> {
+        let active = self.active_capella_org_name().unwrap();
```

Comment:
Can this not panic? If it can't be empty then why does the func return an option?

Reply:
You're right, done now